### PR TITLE
Added linux prereq install script

### PR DESCRIPTION
**Pull request by s1113950**
fixes #{issue_num_here_if_applicable}

If you aren't done, please add the `wip` label until you're ready for review.

If you are adding a new template or mlt command/flag ensure that is is documented in the [docs/features.md](https://github.com/IntelAI/mlt/tree/master/docs/features.md) file.

## Reviewer Checklist

 - [ ] Do you understand it?
 - [ ] Is it correct?
 - [ ] Is it safe?
 - [ ] Is it legally compliant?
 - [ ] Is it robust?
 - [ ] Is it simple?
 - [ ] Is it tested?
 - [ ] Is it documented?
 - [ ] Is the style consistent?

See [the reviewer guide](docs/reviews.md) for more detail on each check.

### CODE REVIEW

**Comment by s1113950** [comment]
🤔 should I just add a mac install script here too? Most of our devs are on macs. @dmsuehir

**Comment by dmsuehir** [comment]
@s1113950 The mac one can be a separate PR if you want.  It's awesome that we can test the linux install script as part of our test cluster setup.  How will we test the mac one?

File: scripts/linux_prereq_install.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#! /bin/bash
+
+# TODO: is this even necessary? Users should have `git` already
+# jq is needed for tensorboard
+# openssh is needed for horovod
+apt-get install git jq openssh-server openssh-client
```
Comment (ashahba) [inline scripts/linux_prereq_install.sh]: Because of this line, should we call the file `ubuntu_prereq_install.sh` ?

**Comment by s1113950** [comment]
Unfortunately it won't be tested regularly :( I'd add a `DISCLAIMER` at the top of the file. We would need a `mac` setup on circleci which would be a paying feature I thought

File: scripts/linux_prereq_install.sh (hunk 1 above)
Comment (s1113950) [reply scripts/linux_prereq_install.sh]: I could rename it to `debian_prereq_install`? `apt` is a debian-based installer :)

Comment (ashahba) [reply scripts/linux_prereq_install.sh]: Sounds good.

Comment (ashahba) [reply scripts/linux_prereq_install.sh]: We could also use this pattern:
https://github.com/IntelAI/mlt/blob/f4775e8a778dfadbd0cb51c6a2c1d79198292a8c/scripts/ksync_install.sh#L26
to make it both `MacOS` and `Linux` compatible, but I don't know if you want to do it for this ticket.

File: Makefile [hunk 2]
```diff
@@ -144,6 +144,10 @@ docker:
 		--build-arg no_proxy=${no_proxy} \
 		-t mlt .
 
+# useful for linux users to get every outside dependency required for mlt to work
+linux_prereq_install:
+	@./scripts/linux_prereq_install.sh
```
Comment (dmsuehir) [inline Makefile]: change to`debian_prereq_install` here too

**Pull request merged**